lib/livebook_web/plugs/auth_plug.ex

```diff
@@ -12,7 +12,18 @@ defmodule LivebookWeb.AuthPlug do
   @impl true
   def call(conn, _opts) do
     if authenticated?(conn) do
-      conn
+      user =
+        LivebookWeb.UserPlug.build_current_user(
+          get_session(conn),
+          conn.assigns.identity_data,
+          conn.assigns.user_data
+        )
```

Comment:
Normally we would need to mirror the logic for LV in `AuthHook`. However, in this case the authorization information (i.e. groups) comes from the initial request and doesn't change when the socket reconnects or on live navigation. So it may be fine to not mirror it. @josevalim please double-check :)

Reply:
@aleDsz well, now that we extracted the logic, I think we can just call `authorized?` here and not worry about it:

https://github.com/livebook-dev/livebook/blob/1628ddbc51f5d4b3993a45efd7f7afc37c0c0580/lib/livebook_web/live/hooks/auth_hook.ex#L9